manim/__main__.py

```diff
@@ -14,7 +15,17 @@ def exit_early(ctx, param, value):
         sys.exit()
 
 
-console.print(f"Manim Community [green]v{__version__}[/green]")
+latest = requests.get(
+    "https://api.github.com/repos/ManimCommunity/manim/releases/latest"
+)
+latest_tag = latest.json()["tag_name"]
+curr_version = "v" + __version__
+
+if latest_tag == curr_version:
+    console.print(f"Manim Community [green]{curr_version}[/green] (latest)\n")
+else:
+    console.print(f"Manim Community [red]{curr_version}[/red] (outdated)")
+    console.print(f"Update available: [green]{latest_tag}[/green]\n")
```

Comment:
I would like to be printed or the request being made only at the end of the animation and it should be disabled using config files or cli options.

Reply:
Would you like this feature to be enabled or disabled by default?

Reply:
Definitely enabled by default -- perhaps the warning message should contain a short blurb about where to disable this settings in the future, but I'm okay with not having a short blurb for that.

Reply:
For people with a slow internet connection:
Would this feature make everytime a web request when the script is running?
And if yes: Would that not result in a longer run time of a script?

Reply:
That's why it should be run at the end of the animation, like how pip is doing. They can disable if they want to.

Reply:
Sounds good. what about the parameter `check_for_new_version = True` at this point:
https://github.com/ManimCommunity/manim/blob/2752ca54d7437371aeecad57302d75189cf7b0de/manim/_config/default.cfg#L16
?

Reply:
Yup, that's what I'm planning to do. I've named it `notify_outdated_version`.